src/frontend/src/btc/components/send/BtcUtxosFee.svelte

```diff
@@ -0,0 +1,36 @@
+<script lang="ts">
+	import { isNullish } from '@dfinity/utils';
+	import { BigNumber } from '@ethersproject/bignumber';
+	import { createEventDispatcher, onMount } from 'svelte';
+	import type { UtxosFee } from '$btc/types/btc-send';
+	import { BTC_DECIMALS } from '$env/tokens.btc.env';
+	import SkeletonText from '$lib/components/ui/SkeletonText.svelte';
+	import Value from '$lib/components/ui/Value.svelte';
+	import { i18n } from '$lib/stores/i18n.store';
+	import { formatToken } from '$lib/utils/format.utils';
+
+	export let utxosFee: UtxosFee | undefined = undefined;
+
+	const dispatch = createEventDispatcher();
+
+	onMount(() => {
+		if (isNullish(utxosFee)) {
+			dispatch('icSelectUtxosFee');
```

Comment:
if we ultimately do not move the related function in this component, what do you think of using the `steps` component in the parent instead of a dispatcher and listener?

Reply:
Moved the logic to `BtcUtxosFee`.